resources/assets/styles/main.scss

```diff
@@ -12,18 +12,27 @@
 @import "~swiper/swiper-bundle.css";
 
 /** Import theme styles */
+@import "common/archive";
 @import "common/global";
 @import "common/typography";
 @import "common/utilities";
 @import "components/buttons";
+@import "components/cards";
 @import "components/comments";
+@import "components/databases";
 @import "components/forms";
-@import "components/wp-classes";
+@import "components/map";
+@import "components/meta";
+@import "components/page-header";
```

Comment:
In order to better separate what's part of slab and shelf, I think it would be better a lot of this to be included in a shelf.scss file, instead of alongside all the slab base styles. 

Another advantage to this is it gives developers a single place to look for defined rules and to make style overrides.